src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -62,8 +61,8 @@
     public static final String DEFAULT_FLEET_STATUS_SERVICE_PUBLISH_TOPIC =
             "$aws/things/{thingName}/greengrassv2/health/json";
     public static final String FLEET_STATUS_TEST_PERIODIC_UPDATE_INTERVAL_SEC = "fssPeriodicUpdateIntervalSec";
-    public static final int DEFAULT_PERIODIC_UPDATE_INTERVAL_SEC = 86_400;
-    static final String FLEET_STATUS_PERIODIC_UPDATE_INTERVAL_SEC = "periodicUpdateIntervalSec";
+    public static final int DEFAULT_PERIODIC_PUBLISH_INTERVAL_SEC = 86_400;
+    public static final String FLEET_STATUS_PERIODIC_PUBLISH_INTERVAL_SEC = "periodicStatusPublishIntervalSeconds";
```

Comment:
You're changing a config key's name, how will the affect current users? The settings won't apply because it will be looking at the wrong name, right?

Reply:
Right. We were not looking at this key. So it won't affect existing customers.

Reply:
How can you say that? It is read on L168 on the left side.

Reply:
We never documented this configuration anywhere. Plus it was looking at the config space of FSS component which I believe will not work. I forget the reason but I believe to have configuration set for components in the Nucleus, we added the system configurations.